@navikt/core/react/src/util/i18n/locales/nb.ts

```diff
@@ -102,4 +102,24 @@ export default {
       reset: "Tilbakestill tidsperspektiv",
     },
   },
+  DatePicker: {
```

Comment:
The translation object name is not ideal since it contains texts for both DatePicker and MonthPicker. Some are shared as well. Should we call it something else?

Reply:
Can't really think of something different that makes sense, so `DatePicker` is probably fine. You are technically picking a date when you pick a month or year, so it still technically works in that sense 🙈